streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorTopology.java

```diff
@@ -167,12 +195,36 @@ public String toString() {
      * @return A string representation of this instance.
      */
     public String toString(final String indent) {
```

Comment:
I prefer we get a newbie ticket to test out this logic in a unit test case.

Reply:
SG